qml/InitDSMonitorDialog.qml

```diff
@@ -363,6 +361,14 @@ Dialog {
         return ipv6re
     }
 
+    /**
+     * Returns an regular expression for Discovery Server GUID
+     */
+    function guidRegex() {
+        var guidRe =  /^(([0-9a-fA-F][0-9a-fA-F]\.){11})([0-9a-fA-F][0-9a-fA-F])$/
```

Comment:
Would not be better like this? Some Fast DDS GUID serialization use also the format `0.` to represent `00.` (I think guid prefix serialization).
```suggestion
        var guidRe =  ^(([0-9a-fA-F]{1,2}\.){11})([0-9a-fA-F]{1,2})$
```

Reply:
**In General**: Please, do not call variable `*Re` (`guidRe`,`ipv4re`), use a more semantic name: `guidRegex`.

Reply:
Addressed in 9fae1c9